packages/api-explorer/src/App.tsx

```diff
@@ -100,17 +100,13 @@ interface SidebarStyleProps {
 const SidebarDivider = styled.div<SidebarStyleProps>`
   transition: border 0.3s;
   border-left: 1px solid
-    ${({ theme, open }) =>
-      open ? theme.colors.palette.charcoal200 : 'transparent'};
+    ${({ theme, open }) => (open ? theme.colors.ui2 : 'transparent')};
   grid-area: divider;
   overflow: visible;
   position: relative;
   &:hover {
     border-left: 1px solid
-      ${({ theme, open }) =>
-        open
-          ? theme.colors.palette.charcoal300
-          : theme.colors.palette.charcoal200};
+      ${({ theme, open }) => (open ? theme.colors.ui3 : theme.colors.ui2)};
```

Comment:
do you want a border on hover if sidebar is closed? its `transparent` above.

you can probably also just set `border-color` here, no need to redeclare the width and type.

Reply:
Adjusted.